tests/waku_relay/crypto_utils.nim

```diff
@@ -0,0 +1,44 @@
+import nimcrypto
+
+
+proc cfbEncode*(key: string, iv: string, data: string): seq[byte] =
```

Comment:
Why is this called `encode` if it actually `encrypts` ? (same for the proc below)

Reply:
These were sourced (and slightly adapted) from `nimcrypto/examples/cfb.nim`, where they used the `encode` and `decode` keywords in the respective variables. Thus, I assumed those keywords would be understood.